multiregion/docs/multiregion.rst

```diff
@@ -378,12 +381,25 @@ You could create all the topics by running the script :devx-examples:`create-top
          Topic: multi-region-default PartitionCount: 1   ReplicationFactor: 4    Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
             Topic: multi-region-default Partition: 0    Leader: 2   Replicas: 2,1,3,4   Isr: 2,1    Offline:    Observers: 3,4
 
+#. View the topic replica placement in Confluent Control Center:
+
+   Navigate to the C3 UI at http://localhost:9021.
+
+   |C3 cluster navigation|
+
+   Click on the "mrc" cluster, then make your way to the "Topics" section.
```

Comment:
It would be helpful to explicitly explain the difference between the 2 clusters

Reply:
the latest docs preview link reflects an explanation of the 2 clusters

Reply:
Awesome, thanks!